### PR TITLE
DDCE-5165: Update to Play 3

### PR DESCRIPTION
**Pull request by MRBeresford**

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -19,27 +19,25 @@ import sbt.*
 
 object AppDependencies {
 
-  val hmrcMongoVersion = "1.3.0"
-  val bootstrapVersion = "7.23.0"
+  val hmrcMongoVersion = "1.8.0"
```
Comment (adageorg) [inline project/AppDependencies.scala:22]: The dependency updates tool reckons we can go to 1.9.0 now.

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -19,27 +19,25 @@ import sbt.*
...
-    "uk.gov.hmrc.mongo"         %% "hmrc-mongo-test-play-28"  % hmrcMongoVersion,
-    "org.scalatest"             %% "scalatest"                % "3.2.17",
-    "org.scalatestplus"         %% "mockito-4-11"             % "3.2.17.0",
-    "com.typesafe.akka"         %% "akka-testkit"             % "2.6.21",
-    "de.leanovate.play-mockws"  %% "play-mockws"              % "2.8.1",
-    "org.wiremock"              % "wiremock-standalone"       % "3.3.1",
+    "uk.gov.hmrc"               %% "bootstrap-test-play-30"   % bootstrapVersion,
+    "uk.gov.hmrc.mongo"         %% "hmrc-mongo-test-play-30"  % hmrcMongoVersion,
+    "org.scalatest"             %% "scalatest"                % "3.2.18",
+    "org.scalatestplus"         %% "mockito-4-11"             % "3.2.18.0",
```
Comment (adageorg) [inline project/AppDependencies.scala:38]: I think most of the others have moved to mockito-5-10 / 3.2.18.0 so unless there's any reason to stay on v4 should we also update this?

Comment (MRBeresford) [reply project/AppDependencies.scala]: moved to mockito-5-10 / 3.2.18.0

File: project/AppDependencies.scala (hunk 1 above)
Comment (MRBeresford) [reply project/AppDependencies.scala]: Moved to `1.9.0`

**Review by adageorg — APPROVED** [review]

**Pull request merged**